src/main/resources/static/javascript/mainApp/admin.controller.js

```diff
@@ -243,6 +243,16 @@
                 _.some($scope.adminsList, { uhUuid: user });
         };
 
+        /**
+         * Helper - addAdmin
+         * Checks if the user is a departmental account
+         * @param {object} user - the user you are checking to see if they are a departmental account
+         * @returns {boolean} true if the user is a departmental account
+         */
+        $scope.isDeptAccount = (user) => {
+            return (user['uid'] === user['uhUuid'] || user['uhUuid'] === "");
+        }
+
```

Comment:
This seems to be repeated code. Maybe move this function into general.controller.js? Then both grouping.controller.js and admin.controller.js have access to this function.

Reply:
And since this is an arrow function, you can drop the `return`:
```
$scope.isDeptAccount = (user) => (user['uid'] === user['uhUuid'] || user['uhUuid'] === "");
```